charter.md

```diff
@@ -4,7 +4,9 @@ The More Instant Messaging Interoperability (MIMI) working group will specify th
 
 Recognizing the need for a standardized security protocol to support group key establishment, authentication, and confidentiality services for messaging, the IETF has specified the Messaging Layer Security (MLS) protocol [I-D.ietf-mls-protocol] and architecture [I-D.ietf-mls-architecture]. MLS is agnostic to the identity system used within any given messaging service; it provides confidentiality of sessions once the participants in a conversation have been identified. To achieve interoperable messaging, the MIMI working group will specify how one or more identity building block technologies (for example, X.509 certificates or Verifiable Credentials) can be used to establish end-to-end cryptographic identity across messaging services, assuming the use of MLS for key establishment. 
 
-Achieving interoperability requires a solution for the introduction problem, i.e., the ability for a user in one application to take an identifier for a target user, determine the application in which that target should be reached, be granted permission to initiate communications, and be able to establish communications with the target. The working group will specify a solution to the introduction problem, together with best practice recommendations for functionality, configuration options, and other aspects. Express user preferences about discoverability and reachability must be respected. 
+Achieving interoperable messaging among MIMI-compliant services requires a solution for the introduction problem, i.e., given an identifier for a target user, a sending user should be able to establish communications with the target after the target user's service has been reached and the sender has been granted permission to initiate communications. The working group will specify a solution to the introduction problem, together with best practice recommendations for functionality, configuration options, and other aspects.
+
+Some messaging services today have features which define how a user is contacted by someone new. Some services require the user's approval while others allow inbound messages from any other user. This group's work will respect the use of these features and allow them to work the same when inbound messages come from an interoperating service.
```

Comment:
I don't know what "respect" means in this case.

Reply:
I think the point is that if the user wants these features to continue working the same in the presence of interoperating services then that should be what happens.